web/client/utils/MapInfoUtils.js

```diff
@@ -211,7 +212,7 @@ const MapInfoUtils = {
         return l.visibility &&
             MapInfoUtils.services[l.type] &&
             (l.queryable === undefined || l.queryable) &&
-            l.group !== "background"
+            l.group !== "background" && l?.featureInfo?.format !== 'HIDDEN'
```

Comment:
This function should be tested to verify that a layer with featureinfo.format === 'hidden' returns false.